docs/getting-started/installation.md

```diff
@@ -2,7 +2,49 @@
 
 ## What is Dokku?
 
-Dokku is an extensible, open source Platform as a Service that runs on a single server of your choice.
+Dokku is an extensible, Open Source Platform as a Service (PaaS) that runs on a single server of your choice.
+
+It makes a job of deploying software to the servers easy.
+You build your application from smaller elements with Docker
+containers, which you wire together by configuring Dokku accordingly.
+In that sense, Dokku is similar to Docker Compose, but more powerful, as it
```

Comment:
I would cut a reference to Docker Compose out - perhaps best not to assume familiarity with other tools in this intro, especially since plenty of dokku users stick with buildpacks. Also not sure if dokku really is "more powerful"... it's subjective and a moving target.

Reply:
@jcrben Fixed.